src/main/util.ts

```diff
@@ -12,14 +12,21 @@ const chalk = require('chalk');
     return path.replace("app.asar", "app.asar.unpacked");
 }
 
+type VideoInfo = {
+    name: string;
+    size: number;
+    mtime: number;
+};
+
 const { exec } = require('child_process');
 const ffmpegPath = fixPathWhenPackaged(require('@ffmpeg-installer/ffmpeg').path);
 const ffprobePath = fixPathWhenPackaged(require('@ffprobe-installer/ffprobe').path);
 const ffmpeg = require('fluent-ffmpeg');
 ffmpeg.setFfmpegPath(ffmpegPath);
 ffmpeg.setFfprobePath(ffprobePath);
-const fs = require('fs');
-const glob = require('glob');
+import fs from 'fs/promises';
+const fscb = require('fs')
```

Comment:
why `fscb`  and not `fsSync` or so? What does cb mean?

Reply:
`fscb` was just chosen because the `fs` module can take a callback (cb) to the non-Sync versions of the fs-calls, like `readdirSync` has a corrosponding `readdir` method, that uses a callback to deliver the results.

So, in turn 'fs' isn't inherently synchronous, onlny the `*Sync` versions of the methods, and that's why I picked that name.